zilencer/management/commands/populate_db.py

```diff
@@ -123,6 +124,39 @@ def subscribe_users_to_streams(realm: Realm, stream_dict: Dict[str, Dict[str, An
     Subscription.objects.bulk_create(subscriptions_to_add)
     RealmAuditLog.objects.bulk_create(all_subscription_logs)
 
+def create_alert_words(realm_id: int) -> None:
+    user_ids = UserProfile.objects.filter(
+        realm_id=realm_id
```

Comment:
`is_bot=False, is_active=True` should probably be in here.

Reply:
fixed